tests/platform/cloud/AcceptanceTests_untrustedSettingsPut.js

```diff
@@ -0,0 +1,265 @@
+    // Succesful use cases that request user settings with proper access tokens granted via Resource Owner GPII Token grant
+    {
+        testDef: {
+            name: "A successful workflow for no authorized solutions",
+
+            // The options below are for sending /access_token request
+            client_id: "Bakersfield-AJC-client-id",
+            client_secret: "Bakersfield-AJC-client-secret",
+            username: gpii.tests.cloud.oauth2.untrustedSettingsPut.key,
+            password: "dummy",
+
+            // The options below are required for sending /untrusted-settings
+            key: gpii.tests.cloud.oauth2.untrustedSettingsPut.key,
+            updatedPrefsSet: gpii.tests.cloud.oauth2.untrustedSettingsPut.updatedPrefsSet,
+            expectedMsg: gpii.flowManager.cloudBased.untrustedSettings.put.messages.success,
+            expectedPrefsSet: gpii.tests.cloud.oauth2.untrustedSettingsPut.expectedPrefsSet
+        },
+        disruptions: [{
+            gradeName: "gpii.tests.cloud.oauth2.untrustedSettingsPut.disruption.mainSequence"
+        }]
+    },
+
+    // Rejected by /untrusted-settings endpoint
+    {
+        testDef: {
+            name: "Attempt to retrieve user settings without providing an access token",
+            key: gpii.tests.cloud.oauth2.untrustedSettingsPut.key
+        },
+        disruptions: [{
+            gradeName: "gpii.tests.cloud.oauth2.untrustedSettingsPut.disruption.untrustedSettingsPutNoAccessTokenSequence",
+            expectedStatusCode: 401
+        }]
+    },
+    {
+        testDef: {
+            name: "Attempt to retrieve user settings by providing a wrong access token",
+            key: gpii.tests.cloud.oauth2.untrustedSettingsPut.key
+        },
+        disruptions: [{
+            gradeName: "gpii.tests.cloud.oauth2.untrustedSettingsPut.disruption.untrustedSettingsPutWrongAccessTokenSequence",
+            expectedStatusCode: 401
+        }]
+    },
+    {
+        testDef: {
+            name: "Attempt to retrieve user settings by providing a key that is not associated with any preferences set",
+
+
+            // The options below are for sending /access_token request
+            client_id: "Bakersfield-AJC-client-id",
+            client_secret: "Bakersfield-AJC-client-secret",
+            username: gpii.tests.cloud.oauth2.untrustedSettingsPut.keyWithoutPrefs,
+            password: "dummy",
+
+            // The options below are required for sending /untrusted-settings
+            key: gpii.tests.cloud.oauth2.untrustedSettingsPut.keyWithoutPrefs,
+            prefsSet: gpii.tests.cloud.oauth2.untrustedSettingsPut.updatedPrefsSet
+        },
+        disruptions: [{
+            gradeName: "gpii.tests.cloud.oauth2.untrustedSettingsPut.disruption.untrustedSettingsPutKeyWithoutPrefs",
+            expectedStatusCode: 404
+        }]
+    }
+];
+
+gpii.tests.cloud.oauth2.untrustedSettingsPut.createTestFile();
```

Comment:
The call to `createTestFile` should be moved to the start of the `mainSequence` to make it symmetrical with `cleanUpTestFile` and to ensure that any other test runs through the `mainSequence` get a fresh instance of the preferences.